src/ntp64.rs

```diff
@@ -102,6 +115,29 @@ impl NTP64 {
     pub fn to_system_time(self) -> SystemTime {
         UNIX_EPOCH + self.to_duration()
     }
+
+    /// Convert to a RFC3339 time representation with nanoseconds precision.
+    /// e.g.: `"2024-07-01T13:51:12.129693000Z"``
+    #[cfg(feature = "std")]
+    pub fn to_string_rfc3339(&self) -> String {
```

Comment:
Should the function be called `to_string_rfc3339_lossy` To reflect the fact the conversion is lossy?